draft-ietf-rtcweb-jsep.xml

```diff
@@ -2276,8 +2276,8 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             description or the current remote description, that m=
             section MUST be recycled by generating an m= section for
             the added RtpTransceiver as if the m= section were being
-            added to the session description, placed at the same index
-            as the m= section with a zero port.</t>
+            added to the session description, including a new MID value,
+            placed at the same index as the m= section with a zero port.</t>
```

Comment:
This seems like it got a little run-on, but I guess I can live with it.

I think I would replace "placed at" with "and placing it at"